### PR TITLE
Create what-do-i-need-to-publish.rst

### PR DESCRIPTION
**Pull request by amy-silcock**
Links need to be researched and made
Would like to add shading to the table. How can we do that?

### CODE REVIEW

**Comment by hayfield** [comment]
> Would like to add shading to the table. How can we do that?

The table will automatically be styled in the same way as other similar tables on the website.

File: en/what-do-i-need-to-publish.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+What do I need to publish?
+=========
+
+Publishing to IATI means you are telling a story about the work your organisation does. For this you break your work down into activities, these could be your programmes or projects. For each activity you detail what the work is, where and when it is happening, who is involved and how it is being financed etc. There are loads of details you can add for each activity. The IATI technical team recommends that you start by adding the details (in the form of elements) below:
```
Comment (hayfield) [inline en/what-do-i-need-to-publish.rst]: `loads` [of details] -> `many` or `lots` or something.

(`loads` is a bit colloquial and doesn't translate cleanly)

File: en/what-do-i-need-to-publish.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+A full table of elements that can be reported in IATI is available here:
+
+* All elements in IATI (either to a web page or just downloads the table)
+
+If you receive funding from DFID, the Dutch or Belgian government you will have additional reporting requirements, for information on these please go to the following pages:
+
+* DFID Requirements (need to find a link)
+
+* [Dutch Requirements](https://www.government.nl/documents/publications/2015/12/01/open-data-and-development-cooperation)
```
Comment (hayfield) [inline en/what-do-i-need-to-publish.rst]: An alternate link style is required for website links, so this should be:

```
`Dutch Requirements <https://www.government.nl/documents/publications/2015/12/01/open-data-and-development-cooperation>`__
```

File: en/what-do-i-need-to-publish.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+| Financial | Budget |  | Yes |
+|  | Transactions |  | Yes |
+
+A full table of elements that can be reported in IATI is available here:
+
+* All elements in IATI (webpage link)
+
+If you receive funding from DFID, the Dutch or Belgian government you will have additional reporting requirements, for information on these please go to the following pages:
+
+* 'Dutch Requirements <https://www.government.nl/documents/publications/2015/12/01/open-data-and-development-cooperation>'_
```
Comment (allthatilk) [inline en/what-do-i-need-to-publish.rst]: Quote marks need to be replaced with backticks and two underscores are needed at the end for external links and consistency with docs elsewhere.

File: en/what-do-i-need-to-publish.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+* 'DFID Requirements <https://www.bond.org.uk/strengthen/iati-ngos>'_
```
Comment (allthatilk) [inline en/what-do-i-need-to-publish.rst]: Same as comment above

File: en/what-do-i-need-to-publish.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+| Classifications | Sector | Yes |
+| Financial | Budget |  | Yes |
+|  | Transactions |  | Yes |
+
+A full table of elements that can be reported in IATI is available here:
+
+* `All elements in IATI <http://iatistandard.org/202/guidance/what-do-i-need-to-publish/list-of-all-elements>`__
```
Comment (andylolz) [inline en/what-do-i-need-to-publish.rst]: Does such a table exist on the standard website?

There’s http://iatistandard.org/202/activity-standard/elements/ … But a) that’s the activity standard, and b) I’m not sure whether it’s okay to link to version-specific stuff in the guidance?

File: en/what-do-i-need-to-publish.rst [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+|Basic Activity Information | Title | Yes |
+|  | Description | Yes |
+|  | Activity Status | Yes |
+|  | Activity Date | Yes |
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+| Classifications | Sector | Yes |
+| Financial | Budget |  | Yes |
+|  | Transactions |  | Yes |
```
Comment (andylolz) [inline en/what-do-i-need-to-publish.rst]: I’m afraid this table isn’t correctly formatted… This looks very close to being correctly formatted for markdown, but the RST table formatting is a bit different.

**Comment by andylolz** [comment]
I don’t think this new page is linked to elsewhere in the guidance. Is that intentional? Or is it intended to be part of the “How to publish” guide?

**Pull request merged**